akka-actor/src/main/scala/akka/actor/ActorRef.scala

```diff
@@ -476,6 +476,47 @@ private[akka] trait MinimalActorRef extends InternalActorRef with LocalRef {
   protected def writeReplace(): AnyRef = SerializedActorRef(this)
 }
 
+/**
+ * An ActorRef that ignores any incoming messages.
+ *
+ * INTERNAL API
+ */
+private[akka] class IgnoreActorRef(systemName: String) extends MinimalActorRef {
+
+  override def provider: ActorRefProvider =
+    throw new UnsupportedOperationException("no provider")
+
+  override def path: ActorPath = IgnoreActorRef.actorPath(systemName)
+
```

Comment:
Might also need `writeReplace` to support Java serialization. See how that is done in for example `akka.actor.Nobody.